src/Injector.ts

```diff
@@ -17,9 +18,13 @@ class Injector {
    */
   injectAndRender (inputPath: string, inputIndexYaml: string, boatsRc: BoatsRC): string {
     const fullPath = upath.join(upath.toUnix(process.cwd()), inputPath);
+
+    const pathFromIndexToBoatsRc = upath.relative('.', upath.dirname(inputIndexYaml));
```

Comment:
Hmm.. There's a comment on where this is used, but not here explaining why this is necessary. Probably should add one.